### PR TITLE
Add API docs

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: doc/dev/source/installation/installation_prod.rst [hunk 1]
```diff
@@ -1,5 +1,8 @@
-Installation guide
-=================================================
+Installation guide (production)
+===============================
+
+.. todo::
+    Update instructions
```
Comment (ThiefMaster) [inline doc/dev/source/installation/installation_prod.rst]: I'd actually remove the outdated instructions for now. Almost all of them are obsolete.

Comment (nop33) [inline doc/dev/source/installation/installation_prod.rst]: Done

File: docs/Makefile [hunk 2]
```diff
@@ -86,9 +89,9 @@ linkcheck:
 	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
 	@echo
 	@echo "Link check complete; look for any errors in the above output " \
-	      "or in $(BUILDDIR)/linkcheck/output.txt."
+		  "or in $(BUILDDIR)/linkcheck/output.txt."
```
Comment (ThiefMaster) [inline docs/Makefile]: What's the reason for these indentation changes? `make` requires tab indentation to indent a block but the spaces look very much intentional to align it nicely.

Comment (nop33) [inline docs/Makefile]: I don't rly remember adding this change

Comment (ThiefMaster) [inline docs/Makefile]: probably your editor reformatting the file

**Pull request merged**